geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -187,6 +187,7 @@
   SET(new SetExecutor(), SUPPORTED, new MinimumParameterRequirements(3)),
   SETNX(new SetNXExecutor(), SUPPORTED, new ExactParameterRequirements(3)),
   STRLEN(new StrlenExecutor(), SUPPORTED, new ExactParameterRequirements(2)),
+  GETSET(new GetSetExecutor(), SUPPORTED, new ExactParameterRequirements(3)),
```

Comment:
Could this be moved to between `GET` and `INCRBY` to preserve the alphabetical ordering of commands?

Reply:
Yes. By the way, I also updated the README file as I had missed that.